UI Rework

TahaKamil opened the pull request:
## Summery

I aim to redo a majority of the UI elements, specifically the way they interact with each other (as there is nothing really wrong with how things look). Alot of the UI is convoluted or overcomplicated, so I aim to remove excess/unnecessary pages and combine the pages with others so that a simple and easy to understand UI is achieved. 

_**Further I will try to add the wow factor, with initial experiments in the create and mood details page.**_

Resolves nothing, since the features/issues have been previously implemented

## Testing

- How should a reviewer test this feature, fix, or UI?
I will try and keep all tests passing but with the UI changes somethings may not work anymore. Furthermore, I will be incrementally working through the pages and hopefully improving them. Here is a list of updated UI's:
	- `None, because I have not started yet`

- Are there any specific commands, steps, or credentials needed?
No, this is UI changes and not functionality changes.

## Merge Instructions

- Any considerations that other PR's may need to take into note? 
I am currently waiting for #225 since it changes the UI for the home feed. So the first update/change will be for the Create Mood Event form and Mood Details page @adityadipakpatel if it is fine with you.

- Should the reviewer delete the branch on merge? N

---

**Reviewer's Checklist**
- [ ] Are all relevant issues referenced by this PR?
- [ ] Does the app still build (locally)?
- [ ] Do all tests pass?
- [ ] Does the feature work as expected?
- [ ] Does this feature preserve the app’s existing stability?  
- [ ] Is the description of the changes correct/present?
- [ ] Have new tests been created or existing tests updated as needed?

Commit:
Empty commit for PR

adityadipakpatel commented:
@TahaKamil 
should i do the fix for the mood details page? or does this PR include that UI fix?

adityadipakpatel commented:
> @TahaKamil should i do the fix for the mood details page? or does this PR include that UI fix?

i meant the "edit mood" and "view profile" button on the same place, but only one of them gets displayed according to the ownership of the mood
if you want, i can just quickly push the mood details page fix in this branch and/or you can take that task if you want to
either is fine for me

let me know by tomorrow afternoon/evening(18 march) if you want the mood details task, else i'm planning to make a new branch and create a PR for the mood details page fix(for the 2 buttons)

TahaKamil commented:
> > @TahaKamil should i do the fix for the mood details page? or does this PR include that UI fix?
> 
> 
> 
> i meant the "edit mood" and "view profile" button on the same place, but only one of them gets displayed according to the ownership of the mood
> 
> if you want, i can just quickly push the mood details page fix in this branch and/or you can take that task if you want to
> 
> either is fine for me
> 
> 
> 
> let me know by tomorrow afternoon/evening(18 march) if you want the mood details task, else i'm planning to make a new branch and create a PR for the mood details page fix(for the 2 buttons)

Yep you can change and update the edit mood and view profile stuff. Currently I have been primarily focused on the create mood and I am working on the profile page or whatever the right most button on the nav bar called. Particularly I am still experimenting with the wow factor (within the create mood form). So go ahead, any helping appreciated!

Commit:
Couple of Changes

adityadipakpatel commented:
@TahaKamil 
i've made a PR for the mood details page changes, once its merged into main, just pull main into your branch using `git pull origin main` from the UI-redo branch


EDIT: view profile button, username changes for now. will complete the edit/delete button changes by tonight

Commit:
Merge branch 'main' of github.com:cmput301-w25/project-kernelcrew into UI-redo

Commit:
Updates to Navigation and Mood Form
Signed-off-by: “Taha <“tkamil@ualberta.ca”>

Changed updated the styles for the forms. Moved the submit button to the
respective fragments and coustimized each form a little.

Commit:
Added Searching and UI changes

The searching is now functional just need to fix mood history page and
tests.

Commit:
Finished Mood History Implementation

Commit:
Search Implmentaion Completed

TahaKamil commented:
Too outdated and not enough improvements were made to justify keeping this.
